python/lsst/ts/externalscripts/maintel/make_lsstcam_calibrations.py

```diff
@@ -0,0 +1,189 @@
+# This file is part of ts_externalscripts
+#
+# Developed for the LSST Telescope and Site Systems.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+
+__all__ = ["MakeLSSTCamCalibrations"]
+
+import yaml
+from lsst.ts.observatory.control import RemoteGroup
+from lsst.ts.observatory.control.maintel.lsstcam import LSSTCam
+
+from ..base_make_calibrations import BaseMakeCalibrations
+
+
+class MakeLSSTCamCalibrations(BaseMakeCalibrations):
+    """Class for taking images, constructing, verifying, and
+    certifying combined calibrations with LSSTCam.
+
+    This class takes bias, darks, and flat exposures with LSSTCam,
+    constructs a combined calibration for each image type by calling
+    the appropiate pipetask via OCPS, and then verifies and certifies
+    each combined calibration. It also optionally produces defects and
+    Photon Transfer Curves.
+    """
+
+    def __init__(self, index=1):
+        super().__init__(
+            index=index,
+            descr="Takes series of bias, darks and flat-field exposures"
+            "with LSSTCam, and constructs combined "
+            "calibrations, verify and certify the results.",
+        )
+        self._lsstcam = LSSTCam(domain=self.domain, log=self.log)
```

Comment:
When you rebase your branch do develop you will notice that now the `BaseMakeCalibrations` script has a new abstract method called `start_remote`. This method is awaited during the configure method and is a better place to initialize the remote. You should do something like:

```
self._lsstcam = None
self._ocps_group = None
```

Here and then do something like:

```
async def start_remotes(self):
    if self._lsstcam is None:
        self._lsstcam = LSSTCam(domain=self.domain, log=self.log)
        await self._lsstcam.start_task

    if self._ocps_group is None:
        self._ocps_group = RemoteGroup(
            domain=self.domain, components=["OCPS:2"], log=self.log
        )
        await self._ocps_group.start_task
```